vue.config.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const path = require('path');
+const serverPort = require('./config/server').port;
+
+const { STORAGE_PATH } = process.env;
+const imagesPath = 'assets/img';
+// const isProduction = NOxDE_ENV === 'production';
+const serverUrl = `http://127.0.0.1:${serverPort}`;
+
+const aliases = {
+  '@': path.resolve(__dirname, './client'),
+  client: '@',
+  assets: '@/assets',
+  components: '@/components',
+  'tce-core': '@/components/common/tce-core',
+  EventBus: '@/EventBus',
+  utils: '@/utils',
+  shared: path.join(__dirname, 'config/shared'),
+  tce: path.join(__dirname, 'content-elements')
+};
+
+const copy = [{ from: 'client/assets/img', to: imagesPath }];
+
+module.exports = {
+  pages: {
+    index: {
+      entry: './client/main.js',
+      title: 'Tailor'
+    }
+  },
+  runtimeCompiler: true,
+  css: {
+    loaderOptions: {
+      scss: {
+        data: '@import "~@/assets/stylesheets/common/_variables.scss";'
+      }
+    }
+  },
+  devServer: {
+    proxy: {
+      '^/api': { target: serverUrl, ws: false },
+      ...STORAGE_PATH && { '^/repository': { target: serverUrl } }
+    }
+    // Override using: `npm run dev:server -- --port <number>`
+  },
```

Comment:
Make sure that you:
1. do not drop type hints
2. copy everything from `poi.config.js`
3. do not alter stuff if no needed

:bowtie: